Evtx/BinaryParser.py

```diff
@@ -186,7 +186,10 @@ def dosdate(dosdate, dostime):
 
 def parse_filetime(qword):
     # see http://integriography.wordpress.com/2010/01/16/using-phython-to-parse-and-present-windows-64-bit-timestamps/
-    return datetime.utcfromtimestamp(float(qword) * 1e-7 - 11644473600)
+    try:
+        return datetime.utcfromtimestamp(float(qword) * 1e-7 - 11644473600)
+    except ValueError:
+        return
```

Comment:
There's two possibilities to keep in mind:
1. The timestamp itself is actually just invalid, for some reason.
2. There's some other bug and we're not parsing the record we think we're parsing.

In the latter case, an exception *should* be raised, because I don't really care about whatever's in the record. But in the former case, I still care about the rest of the record. However, due to the way the APIs are setup, the first place a client could handle the ValueError is in a call to evtx_record_xml_view, which means you lose the entire record.

Adding the wrapper here seemed like a cheap way to keep the invalid timestamp isolated without disrupting further processing. On a second glance, if the invalid filetime originates from a FiletimeTypeNode, then you'd still get an AttributeError when string() is called.

Perhaps we should just return datetime.min?